src/main/java/teammates/logic/core/AccountsLogic.java

```diff
@@ -126,6 +127,7 @@ public InstructorAttributes joinCourseForInstructor(String key, String googleId)
                         .withEmail(instructor.getEmail())
                         .withInstitute(instituteToSave)
                         .withIsInstructor(true)
+                        .withReadNotifications(new ReadNotifications())
```

Comment:
Would it be better to have an empty readNotifications field created by default in the builder?

Reply:
The empty `readNotifications` field is indeed created by default, but I forgot to remove this line. Thanks for the catch!